<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Dataset Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Information</string>
    </property>
    <widget class="QLabel" name="lblWidth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblHeight">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Height :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbOriginalSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Original Size</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Window Parameters</string>
    </property>
    <widget class="QSpinBox" name="sbH">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbW">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Window Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Windows Height</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>dx</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbDx">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbY">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbX">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbDy">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>dy</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>512</width>
      <height>256</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>510</width>
       <height>254</height>
      </rect>
     </property>
     <widget class="QLabel" name="lblView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Original image here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>20</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Frame Parameters</string>
    </property>
    <widget class="QSpinBox" name="sbOutputW">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>38</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbOutputH">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>62</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Height</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbResizeOutput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Check if we want the output size to be different than the window size</string>
     </property>
     <property name="text">
      <string>Resize Output</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbInterpolation">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Interpolation Method</string>
     </property>
     <item>
      <property name="text">
       <string>Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nearest</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cubic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LanczosZ4</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Interpolation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaOutput">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>168</width>
       <height>168</height>
      </rect>
     </property>
     <widget class="QLabel" name="lblViewOutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Processed image here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output class</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm
Config 
&amp;&amp;
Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>120</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Classes</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of classes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbNumClasses">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetLabels">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Labels</string>
     </property>
    </widget>
    <widget class="QListView" name="lstClasses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSkip">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblClasses">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>255</width>
      <height>318</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Files Parameters</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Output name format</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbOutputFormat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.png, 2.png ..</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>classnum_1.png, ..</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>classelabel_1.png, ..</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Output class format</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbOutputClass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Class id (0 or 1 ..)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Class label (&quot;label1&quot; ..)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblOutputFilename">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>161</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>out.txt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOutputFilename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Output images and classes textfile</string>
     </property>
     <property name="text">
      <string>Output filename</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOutputFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Output images and classes textfile</string>
     </property>
     <property name="text">
      <string>Output Folder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOutputFolder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>132</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start naming from&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbCurCase">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="actionLoadFolder"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Parameters</string>
    </property>
    <addaction name="actionImportParameters"/>
    <addaction name="actionExportParameters"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuParameters"/>
  </widget>
  <action name="actionLoadImage">
   <property name="text">
    <string>Load an image</string>
   </property>
  </action>
  <action name="actionLoadFolder">
   <property name="text">
    <string>Select a folder</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionExportParameters">
   <property name="text">
    <string>Export parameters</string>
   </property>
  </action>
  <action name="actionImportParameters">
   <property name="text">
    <string>Import parameters</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>cbOriginalSize</tabstop>
  <tabstop>sbW</tabstop>
  <tabstop>sbH</tabstop>
  <tabstop>sbX</tabstop>
  <tabstop>sbY</tabstop>
  <tabstop>sbDx</tabstop>
  <tabstop>sbDy</tabstop>
  <tabstop>cbResizeOutput</tabstop>
  <tabstop>sbOutputW</tabstop>
  <tabstop>sbOutputH</tabstop>
  <tabstop>btnOutputFolder</tabstop>
  <tabstop>txtOutputFolder</tabstop>
  <tabstop>scrollAreaOutput</tabstop>
  <tabstop>sbNumClasses</tabstop>
  <tabstop>btnSetLabels</tabstop>
  <tabstop>cmbOutputFormat</tabstop>
  <tabstop>cmbOutputClass</tabstop>
  <tabstop>btnOutputFilename</tabstop>
  <tabstop>lstClasses</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnSkip</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
